src/main/java/me/drepic/proton/Proton.java

```diff
@@ -37,15 +38,16 @@ public void onEnable(){
         String host = config.getString("rabbitHost");
         String virtualHost = config.getString("rabbitVirtualHost");
         int port = config.getInt("rabbitPort");
+
+
         try {
             if(config.getBoolean("authorization.useAuthorization")){
                 String user = config.getString("authorization.username");
                 String password = config.getString("authorization.password");
-                manager = new ProtonManager(name, groups, host, virtualHost, port, user, password); //Create manager
+                manager = new RabbitManager(name, groups, host, virtualHost, port, user, password);
             }else{
-                manager = new ProtonManager(name, groups, host, virtualHost, port); //Create manager
+                manager = new RabbitManager(name, groups, host, virtualHost, port, "", "");
```

Comment:
was this constructor removed on purpose?

Reply:
Added back in next commit